internal/painter/gl/shaders.go

```diff
@@ -8,40 +8,70 @@ import "fyne.io/fyne/v2"
 var shaderLineFrag = &fyne.StaticResource{
 	StaticName: "line.frag",
 	StaticContent: []byte(
-		"#version 110\n\nuniform vec4 color;\nuniform float lineWidth;\nuniform float feather;\n\nvarying vec2 delta;\n\nvoid main() {\n    float alpha = color.a;\n    float distance = length(delta);\n\n    if (feather == 0.0 || distance <= lineWidth - feather) {\n        gl_FragColor = color;\n    } else {\n        gl_FragColor = vec4(color.r, color.g, color.b, mix(color.a, 0.0, (distance - (lineWidth - feather)) / feather));\n    }\n}\n"),
+		"#version 110\r\n\r\nuniform vec4 color;\r\nuniform float lineWidth;\r\nuniform float feather;\r\n\r\nvarying vec2 delta;\r\n\r\nvoid main() {\r\n    float alpha = color.a;\r\n    float distance = length(delta);\r\n\r\n    if (feather == 0.0 || distance <= lineWidth - feather) {\r\n        gl_FragColor = color;\r\n    } else {\r\n        gl_FragColor = vec4(color.r, color.g, color.b, mix(color.a, 0.0, (distance - (lineWidth - feather)) / feather));\r\n    }\r\n}\r\n"),
```

Comment:
I think you want to remove all the \r. I don't think they are desirable. Windows is probably causing the weird issues.

Reply:
Removed \r from shaders.go.